### PR TITLE
Patch frontend issues

### PR DESCRIPTION
**Pull request by lakshya-dhariwal**
# Description
- Give keys to maps
- Move API_URL to env so it can be switched from a single place instead of finding it and editing in the code
- export token from one place instead of declaring it in 2 files

### CODE REVIEW

**Comment by lakshya-dhariwal** [comment]
#42

File: client/src/components/Modals/Login.tsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ const Login = ({ setShowModal }: any) => {
           onSubmit={async (data, { setSubmitting }) => {
             const result = await handleLogin(data);
             if (result.success) {
+              console.log(result.token);
```
Comment (shawshankkumar) [inline client/src/components/Modals/Login.tsx]: Remove console log.

File: client/src/utils/api.tsx [hunk 2]
```diff
@@ -32,6 +31,7 @@ export const handleRegister = async (payload: {}): Promise<any> => {
 
 export const handleLogin = async (payload: {}): Promise<any> => {
   try {
+    console.log(API_URL);
```
Comment (shawshankkumar) [inline client/src/utils/api.tsx:34]: Again

File: client/src/components/Modals/Login.tsx [hunk 3]
```diff
@@ -43,7 +43,6 @@ const Login = ({ setShowModal }: any) => {
           onSubmit={async (data, { setSubmitting }) => {
             const result = await handleLogin(data);
             if (result.success) {
-              console.log(result.token);
```
Comment (Srivastava57Harsh) [inline client/src/components/Modals/Login.tsx]: remove console.log

**Review by shawshankkumar — APPROVED** [review]
Looks good.

**Pull request merged**